app/src/main/java/com/example/weatherapp/Data.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.weatherapp
+
+data class Data(
```

Comment:
This class should be named `DayForecast` as specified in the Assignment 3 specification.